exec/csnex/Extensions.cs

```diff
@@ -47,5 +47,51 @@ public static List<Cell> ToCellArray(this List<string> self)
             }
             return r;
         }
+
+        public static bool Compare(this List<Cell> self, List<Cell> rhs)
+        {
+            if (self.Count != rhs.Count) {
+                return false;
+            }
+            for (int i = 0; i < self.Count; i++) {
+                if (!self[i].Equals(rhs[i])) {
+                    return false;
+                }
+            }
+            return true;
+        }
+
+        public static string Quote(this string s)
+        {
+            string r = "\"";
```

Comment:
Since `string` is immutable, for performance here you will want to use `StringBuilder`.

Reply:
Good call.  Set this to be a `StringBuilder()`.